### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by Camilovelag**
## Good day! 💯 

This pull request is intended to add the Views associated with the Models and Controllers.

### 🛠️ Changes made

- [ ] Implement the design from the sneak peek wireframes.
- [ ] Update controllers, views, and requests specs.

I'm expecting feedback.

Happy coding!  👏👏👏

### CODE REVIEW

File: app/views/posts/show.html.erb [hunk 1]
```diff
@@ -1 +1,25 @@
-<h1>Hi! I&apos;m a show an specific post for an specific user page</h1>
\ No newline at end of file
+<main>
```
Comment (MeqdamAlqudah) [inline app/views/posts/show.html.erb:1]: - [x] Good Job the view and the app are working but please make sure that your design follows the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_root_users.png?token=GHSAT0AAAAAAB43NGWUKMN6BZNMAUSFR7ACY6UV6UQ):- 
- first in /users route , please make sure that the background is white as in the design and the width of the  user cards looks exactly like the one given in the design
 
| Your design  | wireframe |
| ------------- | ------------- |
| image  | image  |
- in user page make sure the background is white 
also make sure to add comments to last three posts as in the design.


| Your design  | wireframe |
| ------------- | ------------- |
| image  | image  |
- in the post section make sure that you provide a count for the comments  as in the design and make sure that the background is white to match the design.

| Your design  | wireframe |
| ------------- | ------------- |
| image | image |

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @Camilovelag , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

image

 ## Highlights
:+1: The app is working 
:+1: no linter errors

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/show.html.erb (hunk 1 above)
Comment (Camilovelag) [reply app/views/posts/show.html.erb:1]: Changes done! 😄

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -13,3 +13,86 @@
  *= require_tree .
```
Comment (HAMMAS-SALEEM) [inline app/assets/stylesheets/application.css:13]: ### Your Getting Started Section :red_circle:
image
### Professional Getting Started Section :green_circle:
image

**[Optional]**

- [x] Your Getting Started Section doesn't contain all the required commands to start your application :fearful: Kindly add all the commands to make your readme file descriptive and professional :+1:

Comment (HAMMAS-SALEEM) [inline app/assets/stylesheets/application.css:13]: image
**[Optional]**
- [x] Your issues page link is not working :fearful: Kindly make your link work to make your readme file professional :+1:

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

### Hey @Camilovelag 👋

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Requirements
✔️ Following Linters

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/assets/stylesheets/application.css (hunk 2 above)
Comment (Camilovelag) [reply app/assets/stylesheets/application.css:13]: Thanks! I'll make sure to do that. 😄

Comment (Camilovelag) [reply app/assets/stylesheets/application.css:13]: Thanks! I'll make sure to do that. 😄

**Pull request merged**